autopush/db.py

```diff
@@ -431,6 +431,7 @@ def register_user(self, data):
             exceeds throughput.
 
         """
+        # Fetch a senderid for this user
```

Comment:
The senderID fetch occurs at REST registration for a GCM client. In the code, the fetch occurs in autopush.router.GCMRouter.register (where it's added to both the instance and the router_data for storage), and then reflected back to the client via the new amend_msg() call.